src/coreclr/scripts/superpmi_asmdiffs_checked_release.py

```diff
@@ -0,0 +1,205 @@
+    platform_name = coreclr_args.platform
+
+    # Find the built jit-analyze and put its directory on the PATH
+    jit_analyze_dir = os.path.join(script_dir, "jit-analyze")
+    if not os.path.isdir(jit_analyze_dir):
+        print("Error: jit-analyze not found in {} (continuing)".format(jit_analyze_dir))
+    else:
+        # Put the jit-analyze directory on the PATH so superpmi.py can find it.
+        print("Adding {} to PATH".format(jit_analyze_dir))
+        os.environ["PATH"] = jit_analyze_dir + os.pathsep + os.environ["PATH"]
+
+    # Find the portable `git` installation, and put `git.exe` on the PATH, for use by `jit-analyze`.
+    git_directory = os.path.join(script_dir, "git", "cmd")
+    git_exe_tool = os.path.join(git_directory, "git.exe")
+    if not os.path.isfile(git_exe_tool):
+        print("Error: `git` not found at {} (continuing)".format(git_exe_tool))
+    else:
+        # Put the git/cmd directory on the PATH so jit-analyze can find it.
+        print("Adding {} to PATH".format(git_directory))
+        os.environ["PATH"] = git_directory + os.pathsep + os.environ["PATH"]
+
+    # Figure out which JITs to use
+    os_name = "win" if platform_name.lower() == "windows" else "unix"
+    arch_name = coreclr_args.arch
+    host_arch_name = "x64" if arch_name.endswith("64") else "x86"
+    os_name = "universal" if arch_name.startswith("arm") else os_name
+    base_jit_path = os.path.join(coreclr_args.base_jit_directory, 'clrjit_{}_{}_{}.dll'.format(os_name, arch_name, host_arch_name))
+    diff_jit_path = os.path.join(coreclr_args.diff_jit_directory, 'clrjit_{}_{}_{}.dll'.format(os_name, arch_name, host_arch_name))
+
+    # Find out if it is asmdiffs between Checked binaries and Release binaries
+    diff_with_release = coreclr_args.diff_with_release
+    if diff_with_release is not True:
+        return 1
+
+    # Core_Root is where the superpmi tools (superpmi.exe, mcs.exe) are expected to be found.
+    # We pass the full path of the JITs to use as arguments.
+    core_root_dir = script_dir
+
+    print("Running superpmi.py download to get MCH files")
+
+    log_file = os.path.join(log_directory, "superpmi_download_{}_{}.log".format(platform_name, arch_name))
+    run_command([
+        python_path,
+        os.path.join(script_dir, "superpmi.py"),
+        "download",
+        "--no_progress",
+        "-core_root", core_root_dir,
+        "-target_os", platform_name,
+        "-target_arch", arch_name,
+        "-spmi_location", spmi_location,
+        "-log_level", "debug",
+        "-log_file", log_file
+        ], _exit_on_fail=True)
+
+    print("Running superpmi.py asmdiffs between checked and release binaries")
+    log_file = os.path.join(log_directory, "superpmi_{}_{}.log".format(platform_name, arch_name))
+
+    overall_md_summary_file = os.path.join(spmi_location, "diff_summary.md")
+    if os.path.isfile(overall_md_summary_file):
+        os.remove(overall_md_summary_file)
```

Comment:
There is not going to be a summary file for the Checked/Release case

Reply:
Removed.